site/app/libraries/Core.php

```diff
@@ -173,7 +173,7 @@ public function setSessionManager(SessionManager $manager) {
     private function createEntityManager(AbstractDatabase $database): EntityManager {
         $config = Setup::createAnnotationMetadataConfiguration(
             [FileUtils::joinPaths(__DIR__, '..', 'app', 'entities')],
-            false,
+            $this->config->isDebug(),
```

Comment:
Change makes sense, curious if you knew exactly what turning on debug mode here does versus regular behavior.

Reply:
I forget exactly what I needed it for but after turning it on, things started to work for me. It might have been related to caching. I saw you made a PR for this. I'll remove the changes from this branch and pull master.